common/default.cfg

```diff
@@ -26,8 +26,8 @@ INVALID = "resources/invalid"
 STACK_HASHES = "resources/stack-hashes"
 PATCH_STACK_DEFINITION = "resources/patch-stack-definition.dat"
 FALSE_POSTITIVES = "resources/false-positives"
-PASTA_RESULT = "resources/pasta-result"
-MBOX_RESULT = "resources/mbox-result"
+PASTA_RESULT = "resources/patch-groups"
+MBOX_RESULT = "resources/patch-groups"
```

Comment:
We can consolidate those fields to one field PATCH_GROUPS=...

Reply:
yes, makes sense

Reply:
done!